public/app/features/alerting/unified/hooks/useCombinedRuleNamespaces.ts

```diff
@@ -67,9 +71,13 @@ export function useCombinedRuleNamespaces(rulesSourceName?: string): CombinedRul
     return rulesSources
       .map((rulesSource): CombinedRuleNamespace[] => {
         const rulesSourceName = isCloudRulesSource(rulesSource) ? rulesSource.name : rulesSource;
-        const promRules = promRulesResponses[rulesSourceName]?.result;
+        let promRules = promRulesResponses[rulesSourceName]?.result;
         const rulerRules = rulerRulesResponses[rulesSourceName]?.result;
 
+        if (rulesSourceName === GRAFANA_RULES_SOURCE_NAME && promRuleNamespaces) {
```

Comment:
```suggestion
       
```
if we apply changes above this line is not necessary

Reply:
`rulesSourceName` could be other than `GRAFANA_RULES_SOURCE_NAME` so it's not enough with checking that the `promRuleNamespaces` param is present, we also need to check that we're dealing with `grafana`